osv/ecosystems.py

```diff
@@ -14,15 +14,22 @@
 """Ecosystem helpers."""
 
 import bisect
+import glob
 import json
 import logging
+import os.path
+import subprocess
+import typing
+from abc import ABC, abstractmethod
 
 import packaging.version
 import urllib.parse
 import requests
 
+import osv
```

Comment:
This is causing cyclic-import lint findings and shouldn't be necessary.

Reply:
Resolved in d07cb9c137c3accc689a149838d59e38cbe85527